### PR TITLE
[REF] add break line to feedback hyperlink

### LINKED ISSUE #138: Feedback hyperlink for issue report
Currently, for module template edition has a minimal issue, I think. It is hard (and sometimes confusing) to locate all the to-be-replaced fields when creating a new module, the line below shows the current feedback link

```rst
`here <https://github.com/OCA/{project_repo}/issues/new?body=module:%20{module_name}%0Aversion:%20{version}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

As reStructuredText doesn't keep line break done in raw text and it will be rendered as expected to be. So I suggest something like

```rst
`here <https://github.com/OCA/
{project_repo}
/issues/new?body=module:%20
{module_name}
%0Aversion:%20
{version}
%0A%0A**Steps%20to%20reproduce**%0A%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

### PR DESCRIPTION
**Pull request by osvalr**
add break line for quick edition in feedback hyperlink Fixes #138

### CODE REVIEW

File: template/module/README.rst [hunk 1]
```diff
@@ -51,8 +51,10 @@ Bug Tracker
 
 Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
```
Comment (pedrobaeza) [inline template/module/README.rst]: Maybe you can make the same here

Comment (osvalr) [inline template/module/README.rst]: @pedrobaeza sure, I thought about it

**Comment by nhomar** [comment]
:+1:

**Comment by osvalr** [comment]
Hi @moylop260,

Do you agree with this PR?

**Comment by moylop260** [comment]
:+1:

**Pull request merged**